src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1366,6 +1367,16 @@ public Entity get(HorseJumpEvent evt) {
 				return evt.getEntity();
 			}
 		}, 0);
+		// PlayerTradeEvent
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerTradeEvent")) {
+			EventValues.registerEventValue(PlayerTradeEvent.class, Entity.class, new Getter<Entity, PlayerTradeEvent>() {
+				@Override
+				@Nullable
+				public Entity get(PlayerTradeEvent e) {
+					return e.getVillager();
+				}
+			}, 0);
```

Comment:
```suggestion
			}, EventValues.TIME_NOW);
```

Reply:
Nice! It's much more understandable than just a set of numbers.

Reply:
Indeed. It's fine, there is a PR ready to change them all that's why any additions need to use these values